src/snovault/schema_utils.py

```diff
@@ -250,6 +250,17 @@ def permission(validator, permission, instance, schema):
         yield IgnoreUnchanged(error)
 
 
+orig_uniqueItems = Draft4Validator.VALIDATORS['uniqueItems']
```

Comment:
I would put this up top.  Is there a reason to do it here?

Reply:
If I'm only using something once I prefer to keep its definition close to where it's used.